src/styles/header.scss

```diff
@@ -0,0 +1,72 @@
+.header {
+  margin: 0 auto;
+  width: 100%;
+  background-color: $bg-color;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  position: sticky;
+  z-index: 1000;
+
+  &__container {
+    margin: 0 auto;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 50px;
+  }
+
+  &__logo-link {
+    display: block;
+    height: 40px;
+  }
+}
+
+.nav {
+  &__item {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+
+    &:not(:first-child) {
+      margin-left: 20px;
+    }
+  }
+
+  &__list {
+    display: flex;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__link {
+    display: block;
+    text-decoration: none;
+    padding: 0;
+    margin: 0;
+    font-weight: 500;
+    height: 60px;
+    font-size: 12px;
+    text-transform: uppercase;
+    font-family: $roboto-font;
+    color: $text-color;
+    line-height: $header-size;
+    text-align: center;
+
+    &:hover {
+      color: $accent-color;
+    }
+
+    &.is-active {
+      color: $accent-color;
+    }
+  }
+}
+
+.is-active::after {
+  display: block;
+  content: '';
+  width: 100%;
+  background-color: $accent-color;
+  height: 4px;
+  border-radius: 8px;
+  margin-top: -4px;
```

Comment:
The `margin-top: -4px;` property in `.is-active::after` class seems to be used to position the pseudo-element. However, it would be better to use the `bottom` property for positioning, like `bottom: 0;`, to make the code more understandable.